Cargo.toml

```diff
@@ -37,7 +36,6 @@ members = [
 [patch.crates-io]
 # Oak.
 oak = { path = "sdk/rust/oak" }
-oak_abi = { path = "oak/server/rust/oak_abi" }
```

Comment:
Why not keep this and just redirect it to the new location?

Reply:
Oh! This works! Thanks.